docs/components/query.md

```diff
@@ -260,6 +260,30 @@ If `PathPrefixStrip: /some-path` option or `traefik.frontend.rule.type: PathPref
 Kubernetes Ingress annotation is set, then `Traefik` writes the stripped prefix into X-Forwarded-Prefix header.
 Then, `thanos query --web.prefix-header=X-Forwarded-Prefix` will serve correct HTTP redirects and links prefixed by the stripped path.
 
+For ingress controller nginx, use the flag `--web.prefix-header=X-Forwarded-Prefix` for thanos query
```

Comment:
Which ingress controller are we talking about?
https://github.com/kubernetes/ingress-nginx
or 
https://www.nginx.com/products/nginx/kubernetes-ingress-controller/

?

Just use the correct name ;)